web/components/Feed/Feed.tsx

```diff
@@ -174,7 +179,7 @@ const filterGroups: FilterGroup[] = [
     filters: [
       {
         key: "dogsNotGoodWith",
-        description: "Dogs known to be good with:",
+        description: "Dogs not known to be good with:",
```

Comment:
This filter is for users to see dogs that are confirmed to be good with men, women, etc. I believe we updated the wording but didn't change the key last semester. Consider changing the key instead.